app/js/lib/intent-parser/parsers/action-parser.js

```diff
@@ -0,0 +1,89 @@
+const PATTERNS = {
+  en: {
+    patterns: [
+      /Remind (?:.+?) (?:to|at|on|by|that|about) (.+)/i,
+      /Remind (?:.+?) that (?:it|there) is (.+)/i,
+    ],
+    // @see http://www.unicode.org/cldr/charts/29/summary/en.html#4
+    punctuation: /[-‐–—,;:!?.…'‘’"“”()[\]§@*/&#†‡′″]+$/,
+  },
+
+  fr: {
```

Comment:
I'm going to keep them because I modified them to conform to the new logic. If I remove them, this updated code will be lost.